src/library/Uno.Toolkit.Material/Styles/Controls/v2/CardContentControl.xaml

```diff
@@ -34,7 +28,7 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="utu:CardContentControl">
-                    <Grid x:Name="GridRoot"
+					<Grid x:Name="GridRoot"
```

Comment:
Format
![image](https://github.com/unoplatform/uno.toolkit.ui/assets/16295702/a890fe67-045c-4a53-a32d-ff3b75b73ffe)

Reply:
the tap size is fine
github assume tab=8sp unless told otherwise

Reply:
https://github.com/settings/appearance -> Tab size preference

Reply:
Tab=8sp is crazy, 4 ftw